doc/overview/architecture.md

```diff
@@ -99,43 +99,57 @@ subset of the nodes called storage nodes, have direct access to NVM
-be integrated with different daemon management or orchestration
-frameworks (e.g., a systemd script, a Kubernetes service or even via a
-parallel launcher like pdsh or srun).
-
-A DAOS system is identified by a system name and consists of a set of
-DAOS servers connected to the same fabric. Membership of the DAOS
-servers is recorded into the system map that assigns a unique integer
-rank to each server. Two different systems comprise two disjoint sets of
-servers and do not coordinate with each other.
-
-Inside a DAOS server, the storage is statically partitioned across
-multiple targets to optimize concurrency. To avoid contention, each
-target has its private storage, own pool of service threads and
+### DAOS System, Storage Node, Server, I/O Server, and Target
+
+A DAOS *system* is identified by a system name, and consists of a set of
+DAOS *storage nodes* connected to the same fabric. The DAOS storage nodes
+run one DAOS *server* instance per node, which in turn starts one
+DAOS *I/O server* process per physical socket. Membership of the DAOS
+servers is recorded into the system map, that assigns a unique integer
+*rank* to each *I/O server* process. Two different DAOS systems comprise
+two disjoint sets of DAOS servers, and do not coordinate with each other.
+
+The DAOS *server* is a multi-tenant daemon running on a Linux instance
+(either natively on the physical node or in a VM or container) of each
+*storage node*. Its *I/O server* sub-processes export the locally-attached
+SCM and NVM storage through the network. It listens to a management port
+(addressed by an IP address and a TCP port number), plus one or more fabric
+endpoints (addressed by network URIs).
+The DAOS server is configured through a YAML file in /etc/daos,
+including the configuration of its I/O server sub-processes.
+The DAOS server startup can be integrated with different daemon management or
+orchestration frameworks (for example a systemd script, a Kubernetes service,
+or even via a parallel launcher like pdsh or srun).
+
+Inside a DAOS I/O server, the storage is statically partitioned across
+multiple *targets* to optimize concurrency. To avoid contention, each
+target has its private storage, its own pool of service threads, and its
 dedicated network context that can be directly addressed over the fabric
-independently of the other targets hosted on the same storage node. A
-target is typically associated with a single-ported SCM module and NVMe
-SSD attached to a single storage node. Moreover, a target does not
-implement any internal data protection mechanism against storage media
-failure. As a result, a target is a single point of failure. A dynamic
-state is associated with each target and is set to either up and
-running, or down and not available.
+independently of the other targets hosted on the same storage node.
+The SCM modules are typically configured in *AppDirect interleaved* mode.
```

Comment:
This is the main clarification: A target is **not** associated with a single-ported SCM module, because those are not visible as individual devices when configured in AppDirect interleaved mode. The target uses a fraction of the total capacity of the namespace on the single region / interleave set per socket.